open_humans/templates/socialaccount/signup.html

```diff
@@ -0,0 +1,154 @@
+{% extends 'panel.html' %}
+
+{% load i18n %}
+{% load account socialaccount %}
+{% load utilities %}
+
+
+{% block head_title %}Create an Open Humans account{% endblock %}
+
+{% block panel_content %}
+
+<div class="well">
+  <p>Members must have a <strong>public username and public
```

Comment:
The Balsamiq mock-ups have condensed language that hasn't been implemented here.